news/2024/2024-05-22-osu-keycaps.md

```diff
@@ -0,0 +1,80 @@
+
+Nearly two decades later, you'd be hard-pressed to find an osu! player who doesn't use a mechanical keyboard, so it's about time we stepped up.
+
+## Keycap set
+
+First and foremost, [**this is a keycap set**](https://www.thekapco.com/products/osu-pippi-keycap-set). It comes with everything you'd need to fill a 100% sized keyboard, plus a bunch of extra swappable keys.
+
+[![](LINK)](https://www.thekapco.com/products/osu-pippi-keycap-set)
+
+Keen eyes may have noticed that we replaced `Z` and `X` (osu!'s default input keys) with some *fancy* designs. 
+
+[![](/wiki/shared/news/2024-05-22-osu-keycaps/specialonly.jpg)](https://www.thekapco.com/products/osu-pippi-keycap-set)
+
+Obviously not every circle clicker uses `Z` and `X` though. We've included special keys for 3 different rows, so you can easily swap to match your favourite binds!
+
+[![](/wiki/shared/news/2024-05-22-osu-keycaps/special.jpg)](https://www.thekapco.com/products/osu-pippi-keycap-set)
+
+See it in action:
+
+<div align="center" class="osu-md__paragraph">
+    <video width="75%" controls>
+        <source src="https://assets.ppy.sh/media/news/keycaps.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+(That keycap remover is included too, by the way.)
+
+It's impossible to please both sides of *light mode* vs. *dark mode* debates. Actually, nevermind — we included `Esc` keys for both, designed by [Tanza3D](https://osu.ppy.sh/users/10379965).
+
+[![](/wiki/shared/news/2024-05-22-osu-keycaps/esc.jpg)](https://www.thekapco.com/products/osu-pippi-keycap-set)
+
+Sometimes circle clickers... aren't actually circle clickers, so we've also included an alternative osu!mania spacebar!
+
+[![](/wiki/shared/news/2024-05-22-osu-keycaps/space.jpg)](https://www.thekapco.com/products/osu-pippi-keycap-set)
+
+## Keyboards
+
+If you don't have a compatible keyboard yet, **The KapCo** also offers these keycap sets as proper keyboards too!
+
+Specifically, you can pick up either [this perfectly sized 75% keyboard](https://www.thekapco.com/products/osu-pippi-keycap-set?variant=47259204911337)...
+
+[![](/wiki/shared/news/2024-05-22-osu-keycaps/keyboard.jpg)](https://www.thekapco.com/products/osu-pippi-keycap-set?variant=47259204911337)
+
+...or if that's too much dedication, stay stealthy with [a slick numpad](https://www.thekapco.com/products/osu-pippi-keycap-set?variant=47259204878569).
+
+[![](/wiki/shared/news/2024-05-22-osu-keycaps/numpad.png)](https://www.thekapco.com/products/osu-pippi-keycap-set?variant=47259204878569)
+
+And of course, both of these come with the full keycap sets too!
+
+## Giveaway
+
+*but wait, there's more!*
+
+**The KapCo** was generous enough to give us NUMBER sets of keys for free, so we'll be doing a joint giveaway!
+
+There are two ways to enter:
+
+1. Retweet [our tweet](LINK)
+2. Follow [@thekapco](https://www.instagram.com/thekapco/) on Instagram, and like/comment on [this post](LINK)
```

Comment:
how does this work? are they requiring we do this? how do we choose the winners if it's on their account?

also i dunno how easy it is to get a list of people that retweet these days because x is a pile of dicks.

Reply:
not tied into anything yet. kapco said their goal with the giveaway was to get more instagram follows, but after thinking about it more, i don't think we need to directly provide it, so ill revise this soon

Reply:
hmm, as long as they are okay with this. i'm fine with having the giveaway only be on twitter or instagram (even theirs) but we should choose one location for it to be at, rather than multiple.